HdProduction.HelpDesk.Infrastructure/Services/UserService.cs

```diff
@@ -14,12 +15,20 @@ public UserService(IUserRepository userRepository)
             _userRepository = userRepository;
         }
 
-        public Task CreateAsync(string email, string pwdHash)
+        public async Task<User> CreateAsync(string firstName, string lastName, string email, string pwdHash)
         {
             var pwdHelper = SecurityHelper.Create();
-            _userRepository.Add(new User(email, "abc", "cde", "", 
-                pwdHelper.CreateSaltedPassword(pwdHash), pwdHelper.Salt));
-            return _userRepository.SaveAsync();
+            var user = new User(email, firstName, lastName, "",
+                pwdHelper.CreateSaltedPassword(pwdHash), pwdHelper.Salt);
+            _userRepository.Add(user);
+            await _userRepository.SaveAsync();
+            return user;
+        }
+
+        public async Task<User> Find(long id)
```

Comment:
FindAsync
by convention write Async postfix everywhere in asynchronous methods that you call from code
(controller methods is an exception)